engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java

```diff
@@ -121,13 +121,13 @@
     private Materials materials = new Materials();
     private Buffers buffers = new Buffers();
 
-    ThreadManager threadManager = CoreRegistry.get(ThreadManager.class);
-
     private boolean isTakingScreenshot;
 
     private RenderingConfig renderingConfig = CoreRegistry.get(Config.class).getRendering();
     private RenderingDebugConfig renderingDebugConfig = renderingConfig.getDebug();
 
+    ThreadManager threadManager = CoreRegistry.get(ThreadManager.class);
```

Comment:
Hmmmm... I have the funny feeling that this was in the right place, but should be private. I don't see why it should be package-private. Can you make it private and see if anything explodes?